tools-v2/pkg/cli/command/curvebs/snapshot/copyset/copyset.go

```diff
@@ -56,65 +59,107 @@ func (sRpc *SnapshotRpc) Stub_Func(ctx context.Context) (interface{}, error) {
 	return sRpc.Client.Snapshot(ctx, sRpc.Request)
 }
 
-type SnapshotOneCommand struct {
+type SnapshotAllRpc struct {
```

Comment:
You can split snapshotAllRpc and snapshotRpc into two commands, and then remove these two commands separately, which will make it clearer

Reply:
done